dart/welcome_email/main.dart

```diff
@@ -0,0 +1,108 @@
+import 'dart:convert';
+import 'dart:io';
+import 'package:http/http.dart' as http;
+
+Future<void> start(final req, final res) async {
+  final domain = req.env['MAILGUN_DOMAIN'];
+  final mailgun = MailgunMailer(
+    apiKey: req.env['MAILGUN_API_KEY']!,
+    domain: domain ?? '',
+  );
+
+  // Get the name and email of the newly created user from the payload
+  final payload = jsonDecode(req.payload!);
+  final name = payload['name'];
+  final email = payload['email'];
+
+// Create your email
+  final sent = await mailgun.send(
+      from: 'Welcome to My Awesome App <mailgun@$domain>',
+      to: [email],
+      subject: 'Welcome on board ${name}!',
+      text: 'Hi ${name}\nGreat to have you with us. ! 😍');
+  if (sent) {
+    res.send("email sent successfully.");
+  } else {
+    res.send("failed to send message");
```

Comment:
Let's throw error for failed run. Also, should we have section for errors in README? 🤔 Just an idea.

Reply:
By throwing error mean, we can actually send with relevant status code > 400 ? or we throw exception?
I believe this being executed over HTTP, we can use status code?